app/views/tag/_contributors.html.erb

```diff
@@ -40,19 +40,33 @@
 <% elsif @note_count.nil? || @note_count == 0 %>
   <p><%= raw t('tag.contributors.no_contributors', :tag => params[:id]) %>:</p>
 <% else %>
-  <table class="table">
-    <tr>
-      <th style="width:25%;"><%= t('tag.contributors.contributor') %></th>
-      <th><%= t('tag.contributors.notes') %></th>
-    </tr>
-    <% if @users %>
-      <% @users.each do |user| %>
-        <tr class="users-row">
-          <td><a href='/profile/<%= user.name %>'><i class="fa fa-user"></i> <%= user.name %></a></td>
-          <td><a href='/tag/<%= params[:id] %>/author/<%= user.name %>'><%= t('tag.contributors.notes') %> &raquo;</a></td>
+  <div class="col-md-6">
+    <table class="table">
+      <tr>
+        <th style="width:75%;"><%= t('tag.contributors.people_who\'ve_posted') %></th>
```

Comment:
Hey @IshaGupta18, if you want to change the value, you can do https://github.com/publiclab/plots2/blob/master/config/locales/en.yml#L416 as this way, the translation value will be shown but after failing to find the key in en.yml.

Thanks!

Reply:
So I have done this now. Anything else that can be done @gauravano ?